python/pfs/drp/stella/mergeArms.py

```diff
@@ -39,6 +41,8 @@ class MergeArmsConfig(Config):
     doBarycentricCorr = Field(dtype=bool, default=True, doc="Do barycentric correction?")
     mask = ListField(dtype=str, default=["NO_DATA", "CR", "INTRP", "SAT"],
                      doc="Mask values to reject when combining")
+    pfsConfigFile = Field(dtype=str, default="", doc="""Full pathname of pfsCalib file to use.
+    If of the form "pfsConfig-0x%x-%d.fits", the pfsDesignId and visit0 will be deduced from the filename""")
```

Comment:
If not of that form, the default pfsDesignId and visit0 values should be mentioned in the docString (I believe 666 and 0 respectively).